brayns/engine/scene/ModelManager.cpp

```diff
@@ -70,49 +41,53 @@ class ModelFinder
     }
 
     template<typename Container>
-    static auto &findInstance(Container &instances, const uint32_t instanceId)
+    static auto &find(Container &instances, uint32_t instanceId)
```

Comment:
I would call that get as it returns a reference and use find when the result can be null.